ipynb/energy/EnergyModel_ClusterEnergy.ipynb

```diff
@@ -106,12 +106,12 @@
     "    \"emeter\" : {\n",
     "        'instrument' : 'hwmon',\n",
     "        'conf' : {\n",
-    "            'sites' : [ 'a53', 'a57' ],\n",
+    "            'sites' : [ 'BOARDLITTLE', 'BOARDBIG' ],\n",
```

Comment:
I don't know how hwmon works, but I'm using a Juno r0 with `lsk-3.18-armlt-20161028` and the hwmon setup works with `"board" : 'juno'` and `'sites' : [ 'a53', 'a57' ]`